libsolidity/analysis/DocStringAnalyser.cpp

```diff
@@ -48,9 +48,37 @@ void copyMissingTags(StructurallyDocumentedAnnotation& _target, set<CallableDecl
 	for (auto const& iterator: _target.docTags)
 		existingTags.insert(iterator.first);
 
-	for (auto const& [tag, content]: sourceDoc.docTags)
-		if (tag != "inheritdoc" && !existingTags.count(tag))
-			_target.docTags.emplace(tag, content);
```

Comment:
Do you think it's possible to just do a regex_replace on the content here?

Also not sure why a multimap was chosen for this. Probably a vector of tags would be a better choice here. Tag can be a variant of structs notice, dev, param and return. The return struct can be strings name and description. The number of tags is very small (<20, probably like 5 in actual contracts). Do you think it's worth changing this now?

Reply:
Tags can exist multiple times, like several `@param` tags and such. That's why it's a multimap. If we do such a change I would do it in another PR and not here.. generally I am not opposed to a restructuring of the tags.

Reply:
I'll link https://github.com/ethereum/solidity/issues/8247 for future reference.